frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
-import { useParams, Link } from 'react-router-dom'
+import { useParams, Link, useNavigate } from 'react-router-dom'
 import { API_URL } from '../../constants'
 
 function BookDetails() {
-  const [title, setTitle] = useState(null)
-  const [author, setAuthor] = useState(null)
-  const [description, setDescription] = useState(null)
+  const [book, setBook] = useState(null)
   const { id } = useParams()
+  const navigate = useNavigate()
 
   useEffect(() => {
     const fetchCurrentBook = async () => {
       try {
         const response = await fetch(`${API_URL}/${id}`)
-        console.log('Response status:', response.status);
         if (response.ok) {
```

Comment:
@nshelokar-vf, rather than checking with `response.ok` can we come up with better conditions like `response.success` or any other better alternative?

Reply:
We can use response.status which checks the status code returned by response but there is no method like response.success in fetch api.